proto/outbound.proto

```diff
@@ -7,32 +7,33 @@ option go_package = "github.com/linkerd/linkerd2-proxy-api/go/outbound";
 import "net.proto";
 import "destination.proto";
 import "meta.proto";
+import "grpc_route.proto";
 import "http_route.proto";
 import "google/protobuf/duration.proto";
 
 service OutboundPolicies {
-  rpc Get(TargetSpec) returns (OutboundPolicy) {}
+  rpc Get(TrafficSpec) returns (OutboundPolicy) {}
 
-  rpc Watch(TargetSpec) returns (stream OutboundPolicy) {}
+  rpc Watch(TrafficSpec) returns (stream OutboundPolicy) {}
 }
 
-message TargetSpec {
-  // Identifies the source proxy workload (e.g., pod name).
-  string workload = 1;
-
+message TrafficSpec {
+  // Uniquely identifies the source proxy workload (e.g., pod name) to the
+  // control plane.
+  string source_workload = 1;
 
+  // Describes a target address, as observed by the proxy.
   oneof target {
-    // Target address. This may be the cluster IP of a Kubernetes Service or the
-    // IP of a Pod.
-    io.linkerd.proxy.net.IPAddress address = 2;
+    // Indicates the proxy is connecting to a specific IP:port.
+    io.linkerd.proxy.net.SocketAddress addr = 2;
+
+    // Indicates the proxy is connecting to a named address (like an HTTP
+    // authority).
     string authority = 3;
```

Comment:
Using the word "authority" specifically indicates that it may have a port! Looking around the API, we don't have anywhere else where we use a string name and numeric port, so let's just go with a string encoded address.